packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -277,7 +277,8 @@ Will double processing times. Use only for debugging purposes.",
   },
 
   "chain.archiveDataEpochs": {
-    description: "Number of epochs to retain finalized blobs (minimum of MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS)",
+    description:
+      "Number of epochs to retain finalized blobs (minimum of MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS/MIN_EPOCHS_FOR_DATA_COLUMN_SIDECARS_REQUESTS)",
```

Comment:
in fulu we aren't retaining full blobs anymore, just columns so the flag description still seems a bit off to me

maybe

```suggestion
      "Number of epochs to retain finalized blobs/columns (minimum of MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS/MIN_EPOCHS_FOR_DATA_COLUMN_SIDECARS_REQUESTS)",
```